src/components/home/elements/TokenRecord.vue

```diff
@@ -292,9 +353,12 @@ export default {
                 this.functionIndex = -9
                 if (tokentype === CONTRACT_DESCRIPTOR) {
                     this.functionIndex = SEND_FUNCIDX
+                    this.isSplit = false
```

Comment:
if false is default, this can be picked out.
why only set the send func index?
if this part updated the default functionIndex, how to solve the case of deposit and withdraw case later?

Reply:
know the solution by check the this.split, but why not implement the same idea to send token? and remove the functionIndex parameter here